### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pivovarit**
Corrected some typos as I was going through the README

### CODE REVIEW

**Comment by cangencer** [comment]
verify

File: README.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ Jet is simple to set up. The nodes you start discover each other and
 form a cluster automatically. You can do the same locally, even on the
 same machine (your laptop, for example). This is great for quick testing.
 
-With Jet it's easy to build fault-tolerant and elastic data processing
+With Jet, it's easy to build fault-tolerant and elastic data processing
```
Comment (mtopolnik) [inline README.md]: This comma is not needed.

File: README.md [hunk 2]
```diff
@@ -70,16 +70,16 @@ jet.newJob(p).join();
 
 When you run this, point the `path` variable to some directory with text
 files in it. Jet will analyze all the files and give you the word
-frequency distribution in the log output (for each word it will say how
+frequency distribution in the log output (for each word, it will say how
```
Comment (mtopolnik) [inline README.md]: This comma is not needed.

File: README.md [hunk 3]
```diff
@@ -70,16 +70,16 @@ jet.newJob(p).join();
...
 
 The above was an example of processing data at rest (i.e., _batch
-processing_). It's conceptually simpler than stream processing so we
+processing_). It's conceptually simpler than stream processing, so we
 used it as our first example.
 
 ### Stream Processing with Jet
 
-For stream processing you need a streaming data source. A simple example
+For stream processing, you need a streaming data source. A simple example
```
Comment (mtopolnik) [inline README.md]: This comma is not needed.

File: README.md [hunk 4]
```diff
@@ -103,8 +103,8 @@ p.readFrom(Sources.fileWatcher(path))
 jet.newJob(p).join();
 ```
 
-Before running this make an empty directory and point the `path`
-variable to it. While the job is running copy some text files into it
+Before running this, make an empty directory and point the `path`
```
Comment (mtopolnik) [inline README.md]: This comma is not needed.

Comment (pivovarit) [reply README.md]: Did you verify this with a native?
I'm quite convinced that if you reverse the order of `Make an empty directory and point the `path` variable to it before running this`, it deserves a comma (similar with above examples)

Comment (mtopolnik) [reply README.md]: The difference is whether it's a noun phrase or a subordinate clause. `Before running this` is not a subordinate clause, but `before you run this` would be.

**Pull request closed without merging**